src/transformers/models/qdqbert/tokenization_qdqbert.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
```

Comment:
You don't need to define a new tokenizer if you just re-use the BERT one. In the tokenization auto module, just set a line in the auto mapping that maps "qdqbert" to ("BertTokenizer, "BertTokenizerFast") (there is already an example with ibert using the RoBERTa tokenizer for instance).

Reply:
addressed in commit: https://github.com/huggingface/transformers/pull/14066/commits/873d35220d770907d4af417bdd260071e35a5e28